### ISSUE TITLE
Error while trying to launch the game

### ISSUE THREAD

**Issue by Confusified** [DiFFoZ/LethalPerformance#3]
I cannot launch the game with the mod installed
using version 0.0.3
This is the error being spammed:
```
[Error  : Unity Log] InvalidOperationException: Burst failed to compile the function pointer `Void Test(ReadableViewConstants* ByRef, Int32 ByRef, ReadableShaderVariablesXR*)`
Stack trace:
Unity.Burst.BurstCompiler.Compile (System.Object delegateObj, System.Reflection.MethodInfo methodInfo, System.Boolean isFunctionPointer, System.Boolean isILPostProcessing) (at <367QbHx4fe5DvLAiMy3esiKtTvxDudxuBe>:0)
Unity.Burst.BurstCompiler.Compile (System.Object delegateObj, System.Boolean isFunctionPointer) (at <367QbHx4fe5DvLAiMy3esiKtTvxDudxuBe>:0)
Unity.Burst.BurstCompiler.CompileFunctionPointer[T] (T delegateMethod) (at <367QbHx4fe5DvLAiMy3esiKtTvxDudxuBe>:0)
LethalPerformance.Patches.Patch_HDCamera..cctor () (at _/Patches/Patch_HDCamera.cs:9)
Rethrow as TypeInitializationException: The type initializer for 'LethalPerformance.Patches.Patch_HDCamera' threw an exception.
(wrapper dynamic-method) UnityEngine.Rendering.HighDefinition.HDCamera.DMD<UnityEngine.Rendering.HighDefinition.HDCamera::UpdateShaderVariablesXRCB>(UnityEngine.Rendering.HighDefinition.HDCamera,UnityEngine.Rendering.HighDefinition.ShaderVariablesXR&)
UnityEngine.Rendering.HighDefinition.HDRenderPipeline.UpdateShaderVariablesXRCB (UnityEngine.Rendering.HighDefinition.HDCamera hdCamera, UnityEngine.Rendering.CommandBuffer cmd) (at <b8abf62aeac646cea5c5cde032ff9314>:0)
UnityEngine.Rendering.HighDefinition.HDRenderPipeline.UpdateGlobalConstantBuffers (UnityEngine.Rendering.HighDefinition.HDCamera hdCamera, UnityEngine.Rendering.CommandBuffer cmd) (at <b8abf62aeac646cea5c5cde032ff9314>:0)
UnityEngine.Rendering.HighDefinition.HDRenderPipeline.ExecuteRenderRequest (UnityEngine.Rendering.HighDefinition.HDRenderPipeline+RenderRequest renderRequest, UnityEngine.Rendering.ScriptableRenderContext renderContext, UnityEngine.Rendering.CommandBuffer cmd, UnityEngine.Rendering.HighDefinition.AOVRequestData aovRequest) (at <b8abf62aeac646cea5c5cde032ff9314>:0)
UnityEngine.Rendering.HighDefinition.HDRenderPipeline.Render (UnityEngine.Rendering.ScriptableRenderContext renderContext, System.Collections.Generic.List`1[T] cameras) (at <b8abf62aeac646cea5c5cde032ff9314>:0)
UnityEngine.Rendering.RenderPipeline.InternalRender (UnityEngine.Rendering.ScriptableRenderContext context, System.Collections.Generic.List`1[T] cameras) (at <e27997765c1848b09d8073e5d642717a>:0)
UnityEngine.Rendering.RenderPipelineManager.DoRenderLoop_Internal (UnityEngine.Rendering.RenderPipelineAsset pipe, System.IntPtr loopPtr, UnityEngine.Object renderRequest) (at <e27997765c1848b09d8073e5d642717a>:0)
```

**Comment by DiFFoZ** [comment]
Please send the full log

**Comment by Confusified** [comment]
[Player.log](https://github.com/user-attachments/files/15952950/Player.log)

**Comment by DiFFoZ** [comment]
Should be fixed in 0.0.4, reopen if not :)
